Alpha assembly causes MSBuild failure in ResolveLibraryProjectImports task

I'm attempting to test the new prerelease build `2.1.17-alpha-g5b1e8dff8c` and am currently unable to due to an issue exposed in MSBuild. I've updated the [related issue](https://developercommunity.visualstudio.com/content/problem/164584/cant-build-android-error-msb4018-the-resolvelibrar.html) on the VS site, thought it's closed.

I have two Xamarin projects using the typical structure of `MyProject` as a NET Standard project and `MyProject.Android` for the native parts. If I reference the new PCLCrypto assembly in the base project, it builds fine. However, building the Android project, which references the base project, the build fails.

```
error MSB4018: The "ResolveLibraryProjectImports" task failed unexpectedly.
error MSB4018: System.NotSupportedException: The given path's format is not supported.
error MSB4018:    at System.Security.Permissions.FileIOPermission.EmulateFileIOPermissionChecks(String fullPath)
error MSB4018:    at System.IO.Directory.InternalCreateDirectoryHelper(String path, Boolean checkHost)
error MSB4018:    at System.IO.Directory.CreateDirectory(String path)
error MSB4018:    at Xamarin.Android.Tools.Files.ExtractAll(ZipArchive zip, String destination, Action`2 progressCallback, Func`2 modifyCallback, Func`2 deleteCallback, Boolean forceUpdate)
error MSB4018:    at Xamarin.Android.Tasks.ResolveLibraryProjectImports.Extract(DirectoryAssemblyResolver res, ICollection`1 jars, ICollection`1 resolvedResourceDirectories, ICollection`1 resolvedAssetDirectories, ICollection`1 resolvedEnvironments)
error MSB4018:    at Xamarin.Android.Tasks.ResolveLibraryProjectImports.Execute()
error MSB4018:    at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
```

Running VS 2017 Community 15.6.2. I'm not sure if this is technically your issue. It will make it hard to test for some people, though. Thanks!

--- a/src/Directory.Build.props
+++ b/src/Directory.Build.props
@@ -1,8 +1,8 @@
 <Project>
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <OutputPath>$(MSBuildThisFileDirectory)../bin/$(MSBuildProjectName)/$(Configuration)/</OutputPath>
-    <BaseIntermediateOutputPath>$(MSBuildThisFileDirectory)../obj/$(MSBuildProjectName)/</BaseIntermediateOutputPath>
+    <OutputPath>$(MSBuildThisFileDirectory)..\bin\$(MSBuildProjectName)\$(Configuration)\</OutputPath>
+    <BaseIntermediateOutputPath>$(MSBuildThisFileDirectory)..\obj\$(MSBuildProjectName)\</BaseIntermediateOutputPath>
 
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
 

